metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -355,6 +360,60 @@ def get_datasource_urn_from_id(
             )
         raise ValueError("Could not construct dataset URN")
 
+    def parse_owner_payload(self, payload: Dict[str, Any]) -> Dict[str, str]:
+        owners_dict = {}
+        for owner_data in payload["result"]:
+            email = owner_data["extra"]["email"]
+            value = owner_data["value"]
```

Comment:
Is there any chance that any of these accesses results in a `KeyError` exception?


```suggestion
            email = owner_data.get("extra", {}).get("email", None)
            value = owner_data.get("value", None)
```

Also, what's the `value`? May you set a more descriptive name?

Reply:
value is a unique id preset gives to each user, i can change it to 'id', or should i have both as owner_email, owner_id?